src/components/ProfileView/ProfileCourseList.tsx

```diff
@@ -0,0 +1,137 @@
+'use client';
+
+import React from 'react';
+import { Link, List } from '@mui/material';
```

Comment:
The material UI Link makes the page refresh. It would be better to use the Link from `next/link` here
```suggestion
import { List } from '@mui/material';
import Link from 'next/link';
```

Reply:
Changed it, the page holds now the previous state when closing modal. It would probably be possible to nest the next link and MUI link components, but works this way too.